examples/webgl_loader_wwobj2_parallels.html

```diff
@@ -0,0 +1,414 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - Web Worker Parallel Demo</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+
+		<style>
+			body {
+				font-family: Monospace;
+				background-color: #000;
+				color: #fff;
+				margin: 0px 0px 0px 0px;
+				padding: 0px 0px 0px 0px;
+				border: none;
+				cursor: default;
+			}
+			#info {
+				color: #fff;
+				position: absolute;
+				top: 10px;
+				width: 100%;
+				text-align: center;
+				z-index: 100;
+				display:block;
+			}
+			#info a {
+				color: #f00;
+				font-weight: bold;
+				text-decoration: underline;
+				cursor: pointer
+			}
+			#glFullscreen {
+				width: 100%;
+				height: 100vh;
+				min-width: 640px;
+				min-height: 360px;
+				position: relative;
+				overflow: hidden;
+				z-index: 0;
+			}
+			#example {
+				width: 100%;
+				height: 100%;
+				top: 0px;
+				left: 0px;
+				background-color: #000000;
+			}
+			#feedback {
+				color: darkorange;
+				position: absolute;
+				text-align: left;
+				bottom: 0%;
+				left: 0%;
+				width: auto;
+				padding: 0px 0px 4px 4px;
+			}
+			#dat {
+				user-select: none;
+				position: absolute;
+				left: 0px;
+				top: 0px;
+				color: darkorange;
+				z-Index: 2;
+			}
+		</style>
+	</head>
+
+	<body>
+		<div id="dat">
+
+		</div>
+		<div id="info">
+			<a href="http://threejs.org" target="_blank">three.js</a> - WWOBJLoader2Director Parallels Demo
+		</div>
+		<div id="glFullscreen">
+			<canvas id="example"></canvas>
+		</div>
+		<div id="feedback">
+		</div>
+
+		<script src="js/Detector.js"></script>
+		<script src="../build/three.js"></script>
+		<script src="js/controls/TrackballControls.js"></script>
+		<script src="js/loaders/MTLLoader.js"></script>
+		<script src="js/libs/dat.gui.min.js"></script>
+
+		<script src="js/loaders/OBJLoader2.js"></script>
+		<script src="js/loaders/WWOBJLoader2.js"></script>
+		<script>
+
+			'use strict';
+
+			if ( THREE.examples === undefined ) THREE.examples = {};
```

Comment:
I removed it in the namespace from all examples as it is really not required.

I do the following in `OBJLoader2`:
```
if ( THREE.OBJLoader2 === undefined ) { THREE.OBJLoader2 = {} }
THREE.OBJLoader2.version = '1.0.4';
```
Do you generally object to this practice or just for examples. What do you suggest instead?